cli/dcoscli/data/help/node.txt

```diff
@@ -12,13 +12,24 @@ Usage:
                   [--master-proxy]
                   (--leader | --master | --mesos-id=<mesos-id> | --slave=<slave-id>)
                   [<command>]
+    dcos node snapshot create (<nodes>)...
+    dcos node snapshot delete <snapshot>
+    dcos node snapshot download <snapshot> [--location=<location>]
+    dcos node snapshot (--list | --status | --cancel)
+                       [--json]
 
 Commands:
     log
         Prints the Mesos logs for the leading master node, agent nodes, or both.
     ssh
         Establish an SSH connection to the master or agent nodes of your DC/OS
         cluster.
+    snapshot create
+        Create a snapshot. Nodes can be: ip address, hostname, mesos ID or key words "all", "masters", "agents".
+    snapshot download
+        Download a snapshot.
+    snapshot delete
+        Delete a snapshot.
```

Comment:
@darkonie and @pyronicide, Did we decide to rename these commands to diagnostics?